conf/environments.js

```diff
@@ -33,5 +33,17 @@ module.exports = {
     },
     jasmine: {
         globals: globals.jasmine
+    },
+    phantom: {
+        globals: globals.phantom
+    },
```

Comment:
I don't agree with the reasoning, but it should be `phantomjs` as that's the canonical name: http://phantomjs.org/ (see the logo)

We should probably update the `globals` JSON too then?